chords/when_the_music_fades.onsong

```diff
@@ -5,10 +5,10 @@
 
 Verse 1:
 [D]When the music [A]fades
-All is stripped [Em/G]away
+All is stripped [Em]away
```

Comment:
Worship Together just uses `Em` here; `Em/G` didn't sound right to me (but I could be wrong!)

Reply:
I think it should work with the bass guitar. They can be ignored if needs be.

Reply:
:heavy_check_mark: Reverted!